packages/help-center/src/components/help-center-contact-form.tsx

```diff
@@ -294,7 +303,12 @@ const ContactForm: React.FC< ContactFormProps > = ( { mode, onBackClick, onGoHom
 	};
 
 	const isCTADisabled = () => {
-		return isLoading || ( mode !== 'FORUM' && ! supportSite ) || ! message;
+		return (
+			isLoading ||
+			( mode !== 'FORUM' && ! supportSite ) ||
+			( mode !== 'CHAT' && ! subject ) ||
```

Comment:
Shouldn't this check for `subject` be also for `EMAIL`? From my testing it was also a required field for the endpoint.